lib/src/data/entities/habit.dart

```diff
@@ -6,6 +6,7 @@ class Habit {
   final DateTime startDate;
   final DateTime endDate;
   final String notificationTime;
+  bool isSelected;
```

Comment:
@phenybutum can we remove this field isSelected from data layer and operate with it only on presentation, for example with list of selected habits in home bloc ?